### PR TITLE
Kotti "Görli Classic" Testnet Support

### PR DESCRIPTION
**Pull request by 5chdn**
Kotti, cross-client proof-of-authority testnet for Ethereum Classic.

As per https://github.com/goerli/testnet

- Name: Kotti
- Machine-readable name: `--kotti`
- Stage: launched
  - PoA Engine: `clique`
  - Epoch interval: `30000`
  - Step period: `15`
  - Genesis hash: `0x14c2283285a88fe5fce9bf5c573ab03d6616695d717b12a127188bcacfc743c4`
  - Network ID: `6`
  - Chain ID: `6`
  - Homestead: `0`,
  - EIP150: `0`,
  - EIP155: `0`,
  - Disposal: `0`,
  - ECIP1017EraRounds: `5000000`,
  - EIP160: `0`,
  - ECIP1010Pause: `0`,
  - ECIP1010Length: `2000000`,
- Gitter: https://gitter.im/goerli/kotti/

### CODE REVIEW

**Comment by 5chdn** [comment]
Ok this is ready to be reviewed and tested

`make geth && build/bin/geth --kotti` 

should match:  https://github.com/goerli/testnet/blob/master/geth/kotti.genesis

File: cmd/clef/README.md [hunk 1]
```diff
@@ -212,9 +212,9 @@ None
 #### Result
   - array with account records:
      - account.address [string]: account address that is derived from the generated key
-     - account.type [string]: type of the 
+     - account.type [string]: type of the
```
Comment (whilei) [inline cmd/clef/README.md]: Nitpick, typo. Should be `type of the account`.

**Comment by sorpaas** [comment]
nitpick (but fine to leave it like it is): with `Disposal` being `0`, `ECIP1010` parameters are not used at all.

File: cmd/clef/README.md (hunk 1 above)
Comment (whilei) [reply cmd/clef/README.md]: (fine to leave as-is)

**Review by sorpaas — APPROVED** [review]

**Pull request merged**